modules/backend/layouts/auth.htm

```diff
@@ -10,6 +10,29 @@
         <link rel="icon" type="image/png" href="<?= e(Backend\Models\BrandSetting::getFavicon()) ?>">
         <title><?= e(trans('backend::lang.auth.title')) ?></title>
 
+        <!-- Inline Javascript to reduce lookups -->
+        <script>
+        // Load the Preload Assets	
+        $(document).ready(function(){
+            $(document.body).removeClass('preload')
+
+            $('form input[type=text], form input[type=password]').first().focus()
+        })
+
+        /* Only run on HTTPS connections
+         * Block off Front-end Service Worker from running in the Backend allowing security injections see GitHub #4384
+         */ 
+        if (location.protocol === 'https:') {
+            // Unregister all service workers before signing in to prevent cache issues see GitHub #4088
+            navigator.serviceWorker.getRegistrations().then(
+                function(registrations) {
+                    for (let registration of registrations) {  
+                        registration.unregister();
+                    }
+                });
+            }
```

Comment:
Indentation is screwed up and I don't think this works because jQuery isn't loaded yet. Please don't make PRs without actually testing the changes yourself; even if they seem insignificant